packages/apps/docs/src/components/Layout/components/TreeMenu/Item.tsx

```diff
@@ -25,7 +25,11 @@ export const Item: FC<IItem> = ({ item, level }) => {
   );
 
   return (
-    <li className={classNames(listItemClass, listItemVariants[`l${level}`])}>
+    //TODO: Check with Steven for a good way to do this.
```

Comment:
@sstraatemans I had to add test-id here to be able to identify the element, eslint wasn't too happy about this. Would you be able to support with a slight alternative approach that does not piss off eslint?

Reply:
how about using the `data-testid`? data attributes won't trigger any eslint warnings

Reply:
You're right. Thanks mo! <3